Cargo.toml

```diff
@@ -27,7 +28,8 @@ tracing = "0.1"
 rand = { version = "0.8", optional = true }
 
 [dev-dependencies]
-fuel-tx = { version = "0.13", features = ["random"] }
+fuel-crypto = { version = "0.5", features = ["random"] }
```

Comment:
Was this added as nit? I don't see why we now need `random` from `fuel-crypto`, based in this PR. Besides this, lgtm

Reply:
hmm maybe that was leftover from before I refactored some tests to use the fuel-tx builder